lib/sparse/src/index/inverted_index/inverted_index_ram.rs

```diff
@@ -125,72 +124,21 @@ impl InvertedIndexRam {
     }
 }
 
-/// Builder used in tests to validate `upsert` implementation
-pub struct InvertedIndexBuilder {
-    postings: HashMap<DimId, PostingList>,
-}
-
-impl Default for InvertedIndexBuilder {
-    fn default() -> Self {
-        Self::new()
-    }
-}
-
-impl InvertedIndexBuilder {
```

Comment:
this existing builder was only used for convenience in tests

it required full fledged posting list which is not useful for us here